src/app/modules/design-patterns/decorator/services/math.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class MathService {
+  doCalcFibonacciSequence(num: number, memo: {}) {
```

Comment:
`memo: {}` is not needed here. We gonna create something using decorator. That function should be to calc fibonacci.